galleries/team/metadata.yml

```diff
@@ -1,31 +1,111 @@
 ---
-name: ralf_gommers.png
-title: Ralf Gommers
-github: rgommers
-projects: ['NumPy', 'SciPy', 'PyWavelets']
----
-name: saul_shanabrook.png
-title: Saul Shanabrook
-github: saulshanabrook
-projects: ['JupyterLab', 'metadsl']
----
 name: hameer_abbasi.png
 title: Hameer Abbasi
 github: hameerabbasi
-projects: ['Pydata/Sparse', 'uarray']
+projects: ['NumPy', 'PyData/Sparse', 'uarray']
+---
+name: daniel_althviz.png
+title: Daniel Althviz
+github: dalthviz
+projects: ['Spyder']
+---
+name: peter_bell.png
+title: Peter Bell
+github: peterbell10
+projects: ['SciPy', 'uarray']
+---
+name: james_bourbeau.png
+title: James Bourbeau
+github: jrbourbeau
+projects: ['Dask', 'Numba', 'Zarr']
 ---
 name: carlos_cordoba.png
 title: Carlos Cordoba
 github: ccordoba12
 projects: ['Spyder']
 ---
-name: gonzalo_peña.png
-title: Gonzalo Peña-Castellanos
-github: goanpeca
+name: sameer_deshmukh.png
+title: Sameer Deshmukh
+github: v0dro
+projects: ['XND']
+---
+name: marc_garcia.png
```

Comment:
I sent Ashley a more updated picture for the website. Not sure where did you get this from, but probably better to use the one I sent.

Reply:
I used for the pictures the ones available on the bambooHR website. I will ask Ashley for the one you sent her :+1: